src/Arcus.Scripting.Tests.Integration/Arcus.Scripting.LogicApps.tests.ps1

```diff
@@ -227,5 +227,61 @@ InModuleScope Arcus.Scripting.LogicApps {
                 }
             }
         }
+        Context "Resubmit Failed Logic Apps runs" {
+            It "Resubmit all failed instances for a Logic App"{
```

Comment:
The resubmit functionality also has the ability to resubmit multiple workflows, should we test on that, too?

Reply:
That is a bit harder to do, since I cannot disable the Logic App since the resubmits wont work then, and I don't want to set the schedule of the Logic App every couple of seconds because then the `Get-AzLogicAppRunHistory` will not give me only the resubmitted runs. 

I don't think it is absolutely necessary to create a separate test for multiple failed runs, the code works basically the same for a single run as for multiple.

Reply:
Yes, ok, I understand, good think we have a unit test that checks basic loop functionality. 👍